### ISSUE TITLE
[Supporting the platform-Social Embedded-Create Profile]: Test is not resized when user toggle the large text setting.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#27]
Priority: Low

Repro Steps
Pre-requisite:
Settings -> Accessibility ->tap Font size -> Use the slider to choose your font size

1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Verify the text is resized in large text setting or not.

Expected
""Create Profile"" page should be resize when user select large text setting. Page should adopt system controls in large text setting.

Actual
""Create Profile"" page is not resizing when user select large text setting. Page is not adopting system controls in large text setting.

Note:
1> Issue is repro throughout the application.

MAS Rule: MAS1.4.4

Impact
People with vision impairment and cognitive disabilities will get confused if text is not adopting system controls in large text setting.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool :Large Text Setting
